docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md

```diff
@@ -4,32 +4,33 @@ Nebula Graph supports the following built-in math functions:
-double sin(double x) | Return sine of the argument. |
-double asin(double x) | Return inverse sine of the argument.|
-double cos(double x) | Return cosine of the argument. |
-double acos(double x) | Return inverse cosine of the argument. |
-double tan(double x) | Return tangent of the argument. |
-double atan(double x) | Return inverse tangent the argument. |
-int rand32() | Return a random 32 bit integer. |
-int rand32(int max) | Return a random 32 bit integer in [0, max).  |
-int rand32(int min, int max) | Return a random 32 bit integer in [min, max).|
-int rand64() | Return a random 64 bit integer. |
-int rand64(int max) | Return a random 64 bit integer in [0, max). |
-int rand64(int min, int max) | Return a random 64 bit integer in [min, max).|
-collect() | Put all the collected values to a list.
-collect_set() | Put all the collected values to a set.
-int size() | Return the number of elements in a list.
-map(fun, iter) | Return a map object after applying the given function to each item of a given iterable.
+double abs(double x) | Returns absolute value of the argument. |
+double floor(double x) | Returns the largest integer value smaller than or equal to the argument. (Rounds down)|
+double ceil(double x) | Returns the smallest integer greater than or equal to the argument. (Rounds up) |
+double round(double x) | Returns integral value nearest to the argument, Returnss a number farther away from 0 if the parameter is in the middle.|
+double sqrt(double x) | Returns the square root of the argument. |
+double cbrt(double x) | Returns the cubic root of the argument. |
+double hypot(double x, double y) | Returns the hypotenuse of a right-angled triangle. |
+double pow(double x, double y) | ComPutses the power of the argument. |
+double exp(double x) | Returns the value of e raised to the x power. |
+double exp2(double x) | Returns 2 raised to the argument. |
+double log(double x) | Returns natural logarithm of the argument. |
+double log2(double x) | Returns the base-2 logarithm of the argument. |
+double log10(double x) | Returns the base-10 logarithm of the argument. |
+double sin(double x) | Returns sine of the argument. |
+double asin(double x) | Returns inverse sine of the argument.|
+double cos(double x) | Returns cosine of the argument. |
+double acos(double x) | Returns inverse cosine of the argument. |
+double tan(double x) | Returns tangent of the argument. |
+double atan(double x) | Returns inverse tangent the argument. |
+int rand32() | Returns a random 32 bit integer. |
+int rand32(int max) | Returns a random 32 bit integer in [0, max).  |
+int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
+int rand64() | Returns a random 64 bit integer. |
+int rand64(int max) | Returns a random 64 bit integer in [0, max). |
+int rand64(int min, int max) | Returns a random 64 bit integer in [min, max).|
+collect() | Puts all the collected values to a list.
```

Comment:
better to give examples from cypher collect()/collect_set()/map()/range()

Reply:
I will add later